src/RawMessageIterator.ts

```diff
@@ -9,19 +9,24 @@ export type MessageRow = {
 };
 
 export class RawMessageIterator implements AsyncIterableIterator<RawMessage> {
+  // eslint-disable-next-line @foxglove/prefer-hash-private
```

Comment:
there a reason not to address these now?

Reply:
It technically _could_ be a breaking change since these fields are publicly accessible (if you use ts-ignore or plain JS). However I'll probably end up bumping the major version shortly and will fix it at that time